k8s/plugin/README.md

```diff
@@ -338,3 +338,85 @@ make Rest, Loki and etcd service available outside the cluster. One way to acces
   kubectl port-forward service/mayastor-etcd 2379:2379 -n mayastor
   ```
 </details>
+
+
+<details>
+<summary> Upgrade operations </summary>
+
+```sh
+kubectl-mayastor-install
+`Install` upgrade operator
+
+USAGE:
+    kubectl-mayastor install [OPTIONS] <SUBCOMMAND>
+
+OPTIONS:
+    -h, --help
+            Print help information
+
+    -j, --jaeger <JAEGER>
+            Trace rest requests to the Jaeger endpoint agent
+
+    -k, --kube-config-path <KUBE_CONFIG_PATH>
+            Path to kubeconfig file
+
+    -n, --namespace <NAMESPACE>
+            Kubernetes namespace of mayastor service, defaults to mayastor [default: mayastor]
+
+    -o, --output <OUTPUT>
+            The Output, viz yaml, json [default: none]
+
+    -r, --rest <REST>
+            The rest endpoint to connect to
+
+SUBCOMMANDS:
+    help                Print this message or the help of the given subcommand(s)
+    upgrade-operator    Install, Uninstall upgrade resources
+
+kubectl-mayastor-uninstall
+`Uninstall` upgrade operator
+
+USAGE:
+    kubectl-mayastor uninstall [OPTIONS] <SUBCOMMAND>
+
+OPTIONS:
+    -h, --help
+            Print help information
+
+    -j, --jaeger <JAEGER>
+            Trace rest requests to the Jaeger endpoint agent
+
+    -k, --kube-config-path <KUBE_CONFIG_PATH>
+            Path to kubeconfig file
+
+    -n, --namespace <NAMESPACE>
+            Kubernetes namespace of mayastor service, defaults to mayastor [default: mayastor]
+
+    -o, --output <OUTPUT>
+            The Output, viz yaml, json [default: none]
+
+    -r, --rest <REST>
+            The rest endpoint to connect to
+
+SUBCOMMANDS:
+    help                Print this message or the help of the given subcommand(s)
+    upgrade-operator    Install, Uninstall upgrade resources
+```
+
+**Note**: Each subcommand supports `--help` option to know various other options.
+
+
+**Examples**:
+
+1. To install upgrade resources need for executing upgrade operation
+   ```sh
+   ## Command
+   kubectl mayastor install upgrade-operator -n <mayastor_namespace>
```

Comment:
Guess it depends on the target audience, devs or end users?
If we use `upgrade-operator` then I'm ok with `install upgrade-operator` - but if we use `upgrade`
IMHO `init` is better than `install` because it gets confusing with install upgrade: eg: does this install the upgraded version? And would uninstall upgrade downgrade back?

Reply:
I had a discussion with @avishnu  and then finalised on the below commands .
- kubectl mayastor install upgrade-operator
- kubectl mayastor uninstall upgrade-operator
- kubectl mayastor upgrade
- kubectl mayastor get upgrade-status

Reply:
I had a discussion with @avishnu  and came up with the below commands
- kubectl mayastor install upgrade-operator
- kubectl mayastor uninstall upgrade-operator
- kubectl mayastor upgrade
- kubectl mayastor get upgrade-status